Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessCluster.java

```diff
@@ -813,4 +816,13 @@ public String getYarnNMConnectionUrl() {
         return URI.create(getConnectionUrl()).resolve("yarnui/ws/v1/cluster/apps/").toString();
     }
 
+    @Override
+    public SparkSubmitStorageType getDefaultStorageType() {
+        return SparkSubmitStorageType.ADLA_ACCOUNT_DEFAULT_STORAGE;
```

Comment:
>SparkSubmitStorageType [](start = 15, length = 22)

Supposed to be `DEFAULT_STORAGE_ACCOUNT` rather than `ADLA_ACCOUNT_DEFAULT_STORAGE` here?